identity/src/credentials.ts

```diff
@@ -126,21 +196,61 @@ export const issueHashedCredential = async (
       .digest()
   );
 
-  // generate a verifiableCredential
-  const credential = await _issueCredential(DIDKit, key, expiresInSeconds, {
-    credentialSubject: {
-      "@context": [
-        {
-          hash: "https://schema.org/Text",
-          provider: "https://schema.org/Text",
+  let credential: VerifiableCredential;
+  if (signatureType === "EIP712") {
+    // generate a verifiableCredential
+    credential = await _issueEip712Credential(
+      DIDKit,
+      key,
+      expiresInSeconds,
+      {
+        credentialSubject: {
+          "@context": [
+            {
+              customInfo: "https://schema.org/Thing",
+              hash: "https://schema.org/Text",
+              metaPointer: "https://schema.org/URL",
+              provider: "https://schema.org/Text",
+            },
+          ],
+          // construct a pkh DID on mainnet (:1) for the given wallet address
+          id: `did:pkh:eip155:1:${address}`,
+          provider: record.type,
+          metaPointer:
+            "https://github.com/gitcoinco/passport-scorer/blob/main/api/scorer/settings/gitcoin_passport_weights.py",
+          customInfo: {},
+          hash: `${VERSION}:${hash}`,
         },
-      ],
-      // construct a pkh DID on mainnet (:1) for the given wallet address
-      id: `did:pkh:eip155:1:${address}`,
-      provider: record.type,
-      hash: `${VERSION}:${hash}`,
-    },
-  });
+        // https://www.w3.org/TR/vc-status-list/#statuslist2021entry
```

Comment:
From my understanding this can be added later and should only be included if we would like to a add a revocation status.